larq_compute_engine/core/bgemm_functor.h

```diff
@@ -5,8 +5,6 @@
 #include <cstdint>
 #include <limits>
 
-#include "larq_compute_engine/core/macros.h"
-
```

Comment:
It turned out that we were not using anything in `macros.h` anymore, and for LCE-Micro this gave build errors.